commands/user_test.go

```diff
@@ -52,3 +52,284 @@ func (s *MmctlUnitTestSuite) TestSearchUserCmd() {
 		s.Require().Equal("Unable to find user 'example@example.com'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestUserDeactivateCmd() {
+	s.Run("Deactivate an existing user using email", func() {
+		printer.Clean()
+		emailArg := "example@example.com"
+		mockUser := model.User{Username: "ExampleUser", Email: emailArg}
+
+		s.client.
+			EXPECT().
+			GetUserByEmail(emailArg, "").
+			Return(&mockUser, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			DeleteUser(mockUser.Id).
```

Comment:
Just curious, why do we call `DeleteUser` before calling the `userDeactivateCommand`?

Reply:
Sorry for the late response but I assumed the `userDeactivateCommand` is a wrapper around `DeleteUser` so we don't call it before calling `userDeactivateCommand` but rather during the `userDeactivateCommand`.